pyomo/contrib/trustregion/tests/test_util.py

```diff
@@ -0,0 +1,120 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from io import StringIO
+import sys
+import logging
+
+import pyomo.common.unittest as unittest
+
+from pyomo.contrib.trustregion.util import (
+    IterationLogger, minIgnoreNone, maxIgnoreNone
+)
+from pyomo.common.log import LoggingIntercept
+
+
+class TestLogger(unittest.TestCase):
```

Comment:
This is particularly relevant to the last PR of MindtPy uploading the Logger @ZedongPeng